src/net_processing.h

```diff
@@ -41,6 +41,7 @@ struct CNodeStateStats {
     bool m_addr_relay_enabled{false};
     ServiceFlags their_services;
     int64_t presync_height{-1};
+    int m_misbehavior_score{0};
```

Comment:
```suggestion
    int misbehavior_score{0};
```

nit: I think the `m_` only makes sense if there are methods on the struct/class or if there likely will be.

Reply:
At the moment `developer-notes.md` just says "Class member variables have a `m_` prefix.", and most new variables in this struct allow follow that pattern.